src/all/holonometria/src/eu/kanade/tachiyomi/extension/all/holonometria/Holonometria.kt

```diff
@@ -80,45 +71,17 @@ class Holonometria(
             ?.replace("&amp;", "&")
     }
 
-    override fun chapterListRequest(manga: SManga) =
-        paginatedChapterListRequest(manga.url, 1)
-
-    private fun paginatedChapterListRequest(mangaUrl: String, page: Int) =
-        GET("$baseUrl$mangaUrl".removeSuffix("/") + if (page == 1) "/" else "/page/$page/", headers)
-
-    override fun chapterListParse(response: Response): List<SChapter> {
-        val document = response.asJsoup()
-        val mangaUrl = response.request.url.toString()
-            .substringAfter(baseUrl)
-            .substringBefore("page/")
-
-        val chapters = document.select(chapterListSelector())
-            .map(::chapterFromElement)
-            .toMutableList()
-
-        val lastPage = document.select(".pagenation-list a").last()
-            ?.text()?.toIntOrNull() ?: return chapters
+    override fun chapterListRequest(manga: SManga) = GET("$baseUrl/${manga.url}")
 
-        for (page in 2..lastPage) {
-            val request = paginatedChapterListRequest(mangaUrl, page)
-            val newDocument = client.newCall(request).execute().asJsoup()
+    override fun chapterListSelector() = ".manga-detail__list .manga-detail__list-item"
 
-            val moreChapters = newDocument.select(chapterListSelector())
-                .map(::chapterFromElement)
-
-            chapters.addAll(moreChapters)
-        }
-
-        return chapters
-    }
-
-    override fun chapterListSelector() = "#Archive article"
+    override fun chapterListParse(response: Response): List<SChapter> =
+        super.chapterListParse(response).reversed()
 
     override fun chapterFromElement(element: Element) = SChapter.create().apply {
         setUrlWithoutDomain(element.selectFirst("a")!!.attr("href"))
-        name = element.select(".ttl").text()
-        date_upload = element.selectFirst(".data--date")?.text().parseDate()
-        scanlator = element.selectFirst(".data--category")?.text()
+        name = element.select(".manga-detail__list-title").text()
+        date_upload = element.selectFirst(".manga-detail__list-date")?.text()?.trim().parseDate()
```

Comment:
```suggestion
        date_upload = dateFormat.tryParse(element.selectFirst(".manga-detail__list-date")?.text())
```
need to import, and remove the old `parseDate` function